### PR TITLE
Xlsx imports

### PR DESCRIPTION
**Pull request by norkans7**
Depends on https://github.com/nyaruka/rapidpro/pull/868

### CODE REVIEW

**Comment by ericnewcomer** [comment]
@norkans7 looks like pyexcel supports xls with pyexcel-xls which you've included here. Can we keep the xls-based tests in addition to new xlsx tests?

I think it's okay to export xlsx, but we should still support importing xls.

**Comment by norkans7** [comment]
yes we still import xls files well I just removed the warning only everything is as it was before

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1111,21 +1187,26 @@ def import_csv(cls, task, log=None):
...
+
+        pyexcel.save_as(file_name=out_file.name, dest_file_name=csv_tmp_file)
+
         import_results = dict()
 
         try:
-            contacts = cls.import_xls(open(tmp_file), user, import_params, log, import_results)
-        except XLRDError:
-            contacts = cls.import_raw_csv(open(tmp_file), user, import_params, log, import_results)
+            contacts = cls.import_excel(open(tmp_file), user, import_params, log, import_results)
```
Comment (norkans7) [inline temba/contacts/models.py]: not using the smatmin methods to import, I added a new method so we do not break other project depending on smatmin

File: temba/contacts/tests.py [hunk 2]
```diff
@@ -2639,14 +2654,6 @@ def test_contact_import(self):
                              'The file you provided is missing a required header. At least one of "Phone", "Twitter", '
                              '"Telegram", "Email", "Facebook", "External" should be included.')
 
-        csv_file = open('%s/test_imports/sample_contacts.xlsx' % settings.MEDIA_ROOT, 'rb')
```
Comment (norkans7) [inline temba/contacts/tests.py]: remove the warning of not supported files

**Comment by ericnewcomer** [comment]
It looks like the former tests were changed to xlsx, so we still test for the original xls cases?

**Comment by norkans7** [comment]
those are for exports

**Comment by ericnewcomer** [comment]
Wasn't referring to those ones, but noticed the xls tests are still in there for testing import, was just folded.

**Pull request merged**